client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -408,27 +445,49 @@ export default React.createClass( {
 			);
 		}
 
-		const fullContent = (
+		let fullContent, compactContent, compactNoticeText;
+
+		const editLink = name => paths.domainManagementEdit( this.props.selectedSite.slug, name );
+		if ( severity === 'is-error' ) {
+			fullContent = (
 				<span>
-					{ this.translate( 'Some of your domains may be suspended because your email address is not verified:' ) }
+						{ this.translate( 'Your domains may be suspended because your email address is not verified.' ) }
 					<ul>
 						{ domains.map( ( { name } ) =>
-							<li key={ name }>{ name } <a href={ paths.domainManagementEdit( this.props.selectedSite.slug, name ) }>{ this.translate( 'Fix' ) }</a></li>
+							<li key={ name }>{ name } <a href={ editLink( name ) }>{ this.translate( 'Fix' ) }</a></li>
 						) }
 					</ul>
 				</span>
-			),
-			compactNoticeText = this.translate( 'Issues with your domains.' ),
+			);
+			compactNoticeText = this.translate( 'Issues with your domains.' );
 			compactContent = (
 				<NoticeAction href={ paths.domainManagementList( this.props.selectedSite.slug ) }>
 					{ this.translate( 'Fix' ) }
 				</NoticeAction>
 			);
+		} else if ( severity === 'is-info' ) {
+			fullContent = (
+				<span>
+					{ this.translate( 'Please verify ownership of domains:' ) }
+					<ul>
+						{ domains.map( ( { name } ) =>
+							<li key={ name }>{ name } <a href={ editLink( name ) }>{ this.translate( 'Fix' ) }</a></li>
```

Comment:
I think we should use more unique keys than just the domain name - it's asking for collisions. There are old instances of this pattern in domain warnings, but before we fix those, we shouldn't introduce new ones :grinning: Maybe use something that also points to the purpose/origin of the component? Something like `unverificed-domains-can-manage-${ name }`?